### PR TITLE
Fixed grammer mistakes in README.MD

### PR DESCRIPTION
**Pull request by BabyM0le**
### Your checklist for this pull request
🚨 Please review the [guidelines for contributing](https://github.com/plan-player-analytics/Plan/blob/master/CONTRIBUTING.md#writing-a-good-pull-request) to this repository.

- [x] Make sure your name is added to Contributors file `/Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/html/Contributors.java`
- [x] If PR:ing locale changes also add a LangCode with your name `/Plan/common/src/main/java/com/djrapitops/plan/settings/locale/LangCode.java`

### Description
Fixed grammar mistakes in README.MD.

### CODE REVIEW

File: Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/html/Contributors.java [hunk 1]
```diff
@@ -97,6 +97,7 @@ public class Contributors {
             new Contributor("ahdg6", CODE),
             new Contributor("BratishkaErik", LANG),
             new Contributor("Pingger", CODE)
+            new Contributor("BabyM0le, LANG),
```
Comment (AuroraLS3) [inline Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/html/Contributors.java:100]: Please remove this line.

File: README.md [hunk 2]
```diff
@@ -7,10 +7,9 @@
 
 [API, Documentation & Tutorials](https://github.com/plan-player-analytics/Plan/wiki) | [Releases](https://github.com/plan-player-analytics/Plan/releases) | [DEV builds](https://github.com/plan-player-analytics/Plan/actions/workflows/ci.yml?query=branch%3Amaster) | [Issues & Suggestions](https://github.com/plan-player-analytics/Plan/issues)
 
-Player Analytics is a fully fledged solution for the analytics needs of your minecraft server. A built in webserver displays insights into different aspects on the server such as Online activity, Playerbase and how these change over time.
+Player Analytics is a fully-fledged solution for the analytics needs of your Minecraft server. A built-in web server displays insights into different aspects of the server such as Online activity, Playerbase, and how these change over time.
```
Comment (AuroraLS3) [inline README.md:10]:
```suggestion
Player Analytics is a fully-fledged solution for the analytics needs of your Minecraft server. A built-in webserver displays insights into different aspects of the server such as Online activity, Playerbase, and how these change over time.
```
Webserver is written without space throughout the documentation

**Review by AuroraLS3 — CHANGES_REQUESTED** [review]

**Comment by AuroraLS3** [comment]
grammar* mistakes :)

**Review by AuroraLS3 — APPROVED** [review]
Thanks!

**Comment by BabyM0le** [comment]
No problem, I'm new to GitHub to just trying to learn how it works

**Pull request merged**